src/ui/services/useMetaMask.ts

```diff
@@ -95,15 +109,6 @@ export async function useMetaMaskWalletInfo(): Promise<WalletInfoBackgound | nul
 
 export async function useMetaMaskSignature(message: string): Promise<string | null> {
   const walletInfo = await useMetaMaskWalletInfo();
-  
-  if (walletInfo) {
-    const signer = walletInfo.signer;
-    if (metamaskProviderService) {
-      return await metamaskProviderService?.getSignature(signer, message); 
-    }
 
-    return null;
-  }
-
-  return null;
-} 
+  return walletInfo ? metamaskProviderService?.getSignature(walletInfo.signer, message) ?? null : null;
```

Comment:
*Repeated comment*
`getSignature()` fn is an async fn, since it is calling `Signer.signMessage()` async API, so we need to keep on adding `await`.
https://docs.ethers.org/v4/api-wallet.html#signer-api

Reply:
No need to await for return. It's redundant.
https://eslint.org/docs/latest/rules/no-return-await